RickAndMorty-iOS/Views/Shared/EmptyData.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  EmptyData.swift
+//  RickAndMorty-iOS
+//
+//  Created by Gagan on 2023-03-21.
+//
+import Foundation
+import UIKit
+
+struct EmptyData: Hashable {
+    var id: UUID
+}
+
+private var shimmerLayer: CALayer?
+
+func showLoadingAnimation(currentCell: UICollectionViewCell) {
```

Comment:
Change this two functions to UICollectionViewCell extension.  Instead of currentCell variable use 'self'
there will be two function: 
func showLoadingAnimation()
func hideLoadingAnimation()